src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,23 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 /**
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
```

Comment:
lets extract 'magic' 6 number to the constant:
```suggestion
private static final int FIGURES_COUNT = 6;

public static void main(String[] args) {
        Figure[] figures = new Figure[FIGURES_COUNT];
        FigureSupplier figureSupplier = new FigureSupplier();

        for (int i = 0; i < FIGURES_COUNT; i++) {
```

Reply:
not fixed